pc_helper_online.py

```diff
@@ -9,35 +12,43 @@ def main():
     """
     This code is used only for testing purpose.
     Allowing to prove that Public Cloud related logic is actually working without executing
-    a lot of code which is unrelated to pc_helper
+    a lot of code which is unrelated to pc_helper.
+    As input it getting same folder as bot-ng but processing only two variables which are related to Public Cloud
```

Comment:
Yeah it was not ideal but your version also raising some questions from my side . I tried to improve  mine . If you still don't like it please be more specific than "???" what actually you don't like there

Reply:
`???` typically means that I have no idea what this means :sweat_smile: I wish I could, but that also means that I cannot be more specific.

I gave you a suggestion for rewording, which might reflect what you want to say, but since I couldn't understand what you're saying this is really the best I could come up with.

Reply:
I haven't blindly copy/paste your suggestion but I re-word my version . Is it better now ? Do you understand new version ?